System/lastpass.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+import csv
+# <xbar.title>LastPass</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Jason Rauen</xbar.author>
+# <xbar.author.github>badarsebard</xbar.author.github>
+# <xbar.desc>Display your LastPass vault in the menubar. Utilizes the LastPass CLI tool (https://github.com/lastpass/lastpass-cli).</xbar.desc>
+# <xbar.image>https://www.lastpass.com/-/media/43c6c6862a08410a8ef34ab46a3a750b.ico</xbar.image>
```

Comment:
The image should be a screenshot of your plugin in action, preferably with the menu expanded.

Reply:
Added to commit 3636d95